app/internal/page/certificate_provider_details.go

```diff
@@ -90,25 +86,38 @@ func (d *certificateProviderDetailsForm) Validate() map[string]string {
 	errors := map[string]string{}
 
 	if d.FirstNames == "" {
-		errors["first-names"] = "enterFirstNames"
+		errors["first-names"] = "enterCertificateProviderFirstNames"
 	}
 	if d.LastName == "" {
-		errors["last-name"] = "enterLastName"
+		errors["last-name"] = "enterCertificateProviderLastName"
 	}
-	if d.Email == "" {
-		errors["email"] = "enterEmail"
-	}
-	if d.Dob.Day == "" {
-		errors["date-of-birth"] = "dateOfBirthDay"
-	}
-	if d.Dob.Month == "" {
-		errors["date-of-birth"] = "dateOfBirthMonth"
-	}
-	if d.Dob.Year == "" {
-		errors["date-of-birth"] = "dateOfBirthYear"
+	if d.Dob.Day == "" && d.Dob.Month == "" && d.Dob.Year == "" {
+		errors["date-of-birth"] = "enterCertificateProviderDateOfBirth"
+	} else {
+		if d.Dob.Day == "" {
+			errors["date-of-birth-day"] = "dateOfBirthDay"
```

Comment:
are they going to want these to be "Enter your certificate provider's date of birth's day" (etc.)?